app/src/app.py

```diff
@@ -9,8 +9,8 @@
 
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=["http://localhost"],
-    allow_origin_regex=r"http://localhost:\d+|https://dev-social-benefits-navigator--nava-chatbot-.*\.web\.app",
+    allow_origins=["http://localhost:5173"],
```

Comment:
Let's add a comment like:
```suggestion
    # Imagine LA uses port 5173 for development
    allow_origins=["http://localhost:5173"],
```